configure.py

```diff
@@ -1677,7 +1677,7 @@ def MatchingFor(*versions):
     ActorRel(NonMatching, "d_a_obj_gtaki"),
     ActorRel(NonMatching, "d_a_obj_hami2"),
     ActorRel(NonMatching, "d_a_obj_hami3"),
-    ActorRel(NonMatching, "d_a_obj_hami4"),
+    ActorRel(Matching, "d_a_obj_hami4"),
```

Comment:
Since it matches for all retail versions but not the demo version, you can use `MatchingFor` to mark it as matching for only some versions, like most other TUs in this file